bindings/go/mio/mio.go

```diff
@@ -133,23 +133,33 @@ var threadsN int
 
 // Init initialises mio module.
 // All the usual Motr's init stuff is done here.
-func Init() {
+func Init(connParams ...*string) {
     // Mandatory
-    localEP  := flag.String("ep", "", "my `endpoint` address")
-    haxEP    := flag.String("hax", "", "local hax `endpoint` address")
-    profile  := flag.String("prof", "", "cluster profile `fid`")
-    procFid  := flag.String("proc", "", "my process `fid`")
+    var localEP, haxEP, profile, procFid *string
+    if len(connParams) > 0 && len(connParams) != 4 {
+        log.Panicf("number of connection parameters must be 4 or none")
+    } else if len(connParams) == 4 {
+        localEP = connParams[0]
+        haxEP   = connParams[1]
+        profile = connParams[2]
+        procFid = connParams[3]
+    } else {
+        localEP = flag.String("ep", "", "local `endpoint` address")
+        haxEP   = flag.String("hax", "", "hax `endpoint` address")
+        profile = flag.String("prof", "", "cluster profile `fid`")
+        procFid = flag.String("proc", "", "local process `fid`")
+    }
```

Comment:
Do you think we can remove this?
and all of these parameters must be passed explicitly in `connParams`?

So the `mcp` will have something like this:
```
 60 func main() {
        localEP := flag.String("ep", "", "local `endpoint` address")
        haxEP   := flag.String("hax", "", "hax `endpoint` address")
        profile  := flag.String("prof", "", "cluster profile `fid`")
        procFid := flag.String("proc", "", "local process `fid`")
 61     mio.Init(localEP, haxEP, profile, procFid)
```

Reply:
I'd prefer these parameters to be optional. If the program (like mcp) does not provide them at `mio.Init()` call - it means the library will add them to the mandatory command line parameters automatically. It seems convenient and handy.

Reply:
OK